docs/reference/docs/multi-get.asciidoc

```diff
@@ -97,88 +107,117 @@ GET /_mget
 }
 --------------------------------------------------
 
-
 [float]
-[[mget-fields]]
-==== Fields
+[[mget-routing]]
+===== Routing
 
-Specific stored fields can be specified to be retrieved per document to get, similar to the <<get-stored-fields,stored_fields>> parameter of the Get API.
-For example:
+If routing is used during indexing, the routing value also needs to be specified to retrieve a document. 
+For example, the following request fetches `test/_doc/2`  from the shard corresponding to routing key `key1`,
+and fetches `test/_doc/1` from the shard corresponding to routing key `key2`.
 
 [source,console]
 --------------------------------------------------
-GET /_mget
+GET /_mget?routing=key1
 {
     "docs" : [
         {
             "_index" : "test",
             "_id" : "1",
-            "stored_fields" : ["field1", "field2"]
+            "routing" : "key2"
         },
         {
             "_index" : "test",
-            "_id" : "2",
-            "stored_fields" : ["field3", "field4"]
+            "_id" : "2"
         }
     ]
 }
 --------------------------------------------------
 
-Alternatively, you can specify the `stored_fields` parameter in the query string
-as a default to be applied to all documents.
+[float]
+[[mget-security]]
+===== Security
+
+See <<url-access-control>>.
+
+[float]
+[[multi-get-partial-responses]]
+===== Partial responses
+
+To ensure fast responses, the multi get API responds with partial results if one or more shards fail. 
+See <<shard-failures, Shard failures>> for more information.
+
+[[docs-multi-get-api-path-params]]
+==== {api-path-parms-title}
+
+`<index>`::
+(Optional, string) Name of the index that contains the documents.
+
+[[docs-multi-get-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=preference]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=realtime]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=refresh]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=routing]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=stored_fields]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=source]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=source_excludes]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=source_includes]
+
```

Comment:
I believe we need to include a section for request body parameters here:
https://github.com/elastic/docs/blame/master/shared/api-ref-ex.asciidoc#L102

Reply:
Yep.